livekit-ffi/src/cabi.rs

```diff
@@ -18,20 +20,17 @@ pub type FfiCallbackFn = unsafe extern "C" fn(*const u8, usize);
 pub unsafe extern "C" fn livekit_ffi_initialize(
     cb: FfiCallbackFn,
     capture_logs: bool,
-    sdk: Option<&'static [u8; 16]>,
-    sdk_version: Option<&'static [u8; 16]>,
+    sdk: *const c_char,
+    sdk_version: *const c_char,
 ) {
-    let sdk = std::str::from_utf8(sdk.unwrap()).unwrap().trim_end_matches('\0');
-    let sdk_version = std::str::from_utf8(sdk_version.unwrap()).unwrap().trim_end_matches('\0');
-
     FFI_SERVER.setup(FfiConfig {
         callback_fn: Arc::new(move |event| {
             let data = event.encode_to_vec();
             cb(data.as_ptr(), data.len());
         }),
         capture_logs,
-        sdk: Some(sdk),
-        sdk_version: Some(sdk_version),
+        sdk: CStr::from_ptr(sdk).to_str().ok(),
```

Comment:
this isn't safe, we have no guarantee the pointers will still be alive

Reply:
Makes sense - I'm not too familiar with this but cursor suggested  using `.to_string_lossy().into_owned()`:

```
sdk: CStr::from_ptr(sdk).to_string_lossy().into_owned(),
sdk_version: Some(CStr::from_ptr(sdk_version).to_string_lossy().into_owned()),
```

does that make sense?